moldesign/interfaces/biopython_interface.py

```diff
@@ -24,6 +24,12 @@
 from moldesign.helpers.pdb import BioAssembly
 
 
+def exports(o):
```

Comment:
Can you explain how this works?  Just curious as I see it's used in many different classes and my Python understanding is failing me.